examples/01_tutorials/plot_first_level_model_details.py

```diff
@@ -195,11 +195,17 @@ def plot_contrast(first_level_model):
 # Changing the drift model
 # ------------------------
 # 
-# By default the drift model is a set of slow oscillating functions (Discrete Cosine transform), with a cutoff at frequency 1/128 hz.
-# We can change this cut-off, e.g. to 1/64Hz.
-# This is done by setting period_cut=64(s)
-
-first_level_model = FirstLevelModel(t_r, period_cut=64)
+# The drift model is a set of slow oscillating functions
+# (Discrete Cosine transform) with a cut-off frequency. To remove
+# spurious low-frequency effects related to heart rate, breathing and
+# slow drifts in the scanner signal, the standard cutoff frequency
+# is 1/128 Hz. Depending on the design of the experiment, the user may
+# want to change this value. The cutoff period (1/high_pass) should be
+# set as the longest period between two trials of the same condition
+# multiplied by 2. For instance, if the longest period is 32s,
+# the high_pass frequency shall be 1/64 Hz ~ 0.016 Hz.
```

Comment:
Correct, but I prefer to recommend .01 as a baseline. Maybe simply add that .01 is a standard paradigm-independent value.

Reply:
Thx. I've addressed this point [in my latest commit](https://github.com/nistats/nistats/pull/351/commits/b80487a30e8bc523592345d953571c68ed2d76c7).